vividus-plugin-web-app/src/test/java/org/vividus/selenium/screenshot/FilesystemScreenshotDebuggerTests.java

```diff
@@ -57,6 +62,27 @@ void shouldNotSaveDebugScreenshotIfFolderNotSpecified()
         assertThat(testLogger.getLoggingEvents(), empty());
     }
 
+    @Test
+    void shouldCleanUpFolder(@TempDir File debugFolder) throws IOException
+    {
+        filesystemScreenshotDebugger.setDebugScreenshotsLocation(Optional.of(debugFolder));
+        assertThat(debugFolder.list(), is(arrayWithSize(0)));
+        assertTrue(new File(debugFolder, "temp.txt").createNewFile());
+        assertThat(debugFolder.list(), is(arrayWithSize(1)));
+        filesystemScreenshotDebugger.cleanUp();
+        assertThat(debugFolder.list(), is(arrayWithSize(0)));
+    }
+
+    @Test
+    @SuppressFBWarnings("DMI_HARDCODED_ABSOLUTE_FILENAME")
+    void shouldLogExceptionIfCleanUpFails() throws IOException
+    {
+        File invalidFolder = new File("/invalid");
```

Comment:
even for invalid folders I recommend to use `@TempDir`, since we don't know all environment conditions and interaction with file system is always fragile